.github/workflows/iroha2-deploy-base-image.yml

```diff
@@ -0,0 +1,81 @@
+name: I2::Base Image Deploy
+
```

Comment:
We should have a way of pushing the iroha2-ci:dev image before merging into iroha2-dev.

Reply:
created issue

Reply:
DevOps team got creds for the user that has write access to Hyperledger DockerHub account.
[iroha2-ci:dev image](https://hub.docker.com/layers/iroha2-ci/hyperledger/iroha2-ci/dev/images/sha256-776f52d9c1247250c84c3c00b8f2465146e65571fe6fc76177db572f6d1bde30?context=explore)